### ISSUE TITLE
Delete PayoutMethod Front-End

### ISSUE THREAD

**Issue by amihaiemil** [self-xdsd/self-web#387]
On the Contributor Dashboard page, Payout Methods tab, we should allow the removal of the Stripe PayoutMethod. 

Backend endpoint: ``DELETE /api/contributor/payoutmethods/stripe``
Response codes: 204 NO CONTENT if the deletion was successful and BAD REQUEST or SERVER error (with JSON message) in other cases. 

```json
{
  "message": "Something went wrong"
}
```

In case of an unsuccessful response (!= NO CONTENT), we should simply display the ``message`` as a red error on the page, under the buttons.

Should look like this:

<img width="608" alt="Screenshot 2021-03-03 at 09 46 47" src="https://user-images.githubusercontent.com/6305156/109771394-6bc9cc80-7c05-11eb-9249-d10674cc14f4.png">

The ``Delete`` button should be right next to the ``Stripe Dashboard`` button, in its own ``<form>`` element with it's own ``submit(...)`` handler. Don't forget to disable both buttons while the ajax call is executing.

**Comment by zoeself** [comment]
@Lumi3011 this is your task now, please go ahead. Deadline (when this ticket should be closed) is ``2021-03-13T07:27:18.417326``.

Estimation here is ``180 minutes``, that's how much you will be paid.

Remember, you don't have to solve everything in this ticket - you can solve it [partially](https://docs.self-xdsd.com/microtasking.html) and leave ``todo`` markers in the code, which will become future tasks.

If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.

If you don't have time or simply don't want to work on this, you can always [resign](https://docs.self-xdsd.com/projectmanager.html#commands).

**Comment by zoeself** [comment]
@amihaiemil please keep in mind that manual assignment of tickets is a bad practice and it is discouraged. Next time, please let me elect the assignee.

**Comment by amihaiemil** [comment]
@Lumi3011 also, please run the following two ALTER statements on your DB, I don't think you have these updates:

```sql
ALTER TABLE `self_xdsd`.`slf_payoutmethods_xdsd` DROP COLUMN `active`;
ALTER TABLE `self_xdsd`.`slf_invoices_xdsd` ADD COLUMN `billedByCountry` VARCHAR(32) NULL DEFAULT NULL, ADD COLUMN `billedToCountry` VARCHAR(32) NULL DEFAULT NULL, ADD COLUMN `eurToRon` DECIMAL(20,0) NULL DEFAULT 0;
```

**Comment by zoeself** [comment]
@Lumi3011 Don't forget to close this ticket before the deadline (2021-03-13T07:27:18). You are past the first half of the allowed period.

**Comment by zoeself** [comment]
@Lumi3011 Looks like you've missed the task deadline (2021-03-13T07:27:18). You are now resigned from this task.

Please stop working on it, you will not be paid. I will assign it to someone else soon.

**Comment by zoeself** [comment]
@criske this is your task now, please go ahead. Deadline (when this ticket should be closed) is ``2021-03-23T13:27:31.594809``.

Estimation here is ``180 minutes``, that's how much you will be paid.

Remember, you don't have to solve everything in this ticket - you can solve it [partially](https://docs.self-xdsd.com/microtasking.html) and leave ``todo`` markers in the code, which will become future tasks.

If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.

If you don't have time or simply don't want to work on this, you can always [resign](https://docs.self-xdsd.com/projectmanager.html#commands).

**Comment by amihaiemil** [comment]
@zoeself remove

**Comment by zoeself** [comment]
> @zoeself remove

@amihaiemil ok, I've removed this task from scope. I'm not managing it anymore.

**Comment by amihaiemil** [comment]
@zoeself add

**Comment by zoeself** [comment]
> @zoeself add

@amihaiemil I've just registered this ticket as a task and will assign it to someone soon. Thanks!

**Comment by zoeself** [comment]
@Lumi3011 this is your task now, please go ahead. Deadline (when this ticket should be closed) is ``2021-03-23T15:17:17.730277``.

Estimation here is ``180 minutes``, that's how much you will be paid.

Remember, you don't have to solve everything in this ticket - you can solve it [partially](https://docs.self-xdsd.com/microtasking.html) and leave ``todo`` markers in the code, which will become future tasks.

If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.

If you don't have time or simply don't want to work on this, you can always [resign](https://docs.self-xdsd.com/projectmanager.html#commands).

**Comment by zoeself** [comment]
@Lumi3011 thank you for resolving this ticket. I've just added it to your active invoice. You can always check all your invoices and more on the [Contributor Dashboard](https://self-xdsd.com).
